src/ansys/dpf/post/result_workflows/_connect_workflow_inputs.py

```diff
@@ -79,14 +84,36 @@ def _connect_initial_results_inputs(
     mesh: MeshedRegion,
     streams_provider: Any,
     data_sources: Any,
+    averaging_config: AveragingConfig,
 ):
     """Connects the inputs of the initial result workflow.
 
     The initial result workflow is the first workflow in the result workflows chain, which
     extracts the raw results from the data sources.
     """
+    selection_wf = selection.spatial_selection._selection
+
+    if selection.spatial_selection.requires_mesh:
+        selection_wf.connect(_WfNames.initial_mesh, mesh)
+
+    if averaging_config.average_per_body:
+        _enrich_mesh_with_property_fields(
```

Comment:
I thought that the Operator splitting the mesh with a scoping had an option for that?

Reply:
I don't see anything related in the docs: https://dpf.docs.pyansys.com/version/stable/operator_reference_load.html

Reply:
No its scoping::by_property. We split the scoping and average the results separately. But all the results have still the original mesh as a support.